compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/Types.java

```diff
@@ -2756,6 +2788,61 @@ BType getTypeForUnionTypeMembersAssignableToType(BUnionType unionType, BType tar
         }
     }
 
+    BType getTypeForMapTypeMembersAssignableToType(BMapType mapType, BType targetType) {
```

Comment:
Are these names accurate? Just gets the intersection, right?

Reply:
Changed